addons/mail/static/src/js/systray.js

```diff
@@ -138,14 +123,107 @@ var MessagingMenu = Widget.extend({
     },
     on_click_channel: function (event) {
```

Comment:
move this function to the Handlers section, rename it to _onClickChannel, and add docstring